crypto/x509v3/v3_utl.c

```diff
@@ -100,6 +101,45 @@ int X509V3_add_value_bool_nf(const char *name, int asn1_bool,
     return 1;
 }
 
+static char *bignum_to_string(const BIGNUM *bn)
+{
+    char *tmp, *ret;
+    size_t len;
+
+    /*
+     * Display large numbers in hex and small numbers in decimal. Converting to
+     * decimal takes quadratic time and is no more useful than hex for large
+     * numbers.
+     */
+    if (BN_num_bits(bn) < 128) {
```

Comment:
gah.  boring always uses curlies, we avoid it when there's just a single simple statement.  i can fix when merging i suppose.

Reply:
Ah yes, my bad. I always forget we differ on that one.